packages/components/molecules/f-media-element/src/components/MediaElement.vue

```diff
@@ -18,11 +19,12 @@
             <p :class="$style['c-mediaElement-text']">
                 {{ text }}
             </p>
-            <slot></slot>
+            <slot />
         </div>
         <div
             :class="[
                 $style['c-mediaElement-imgWrapper'],
+                'c-mediaElement-imgWrapper--override',
```

Comment:
What does this class do? Can't see a reference to it in the SCSS

Reply:
This allows the user who uses this component to override the imageWrapper class, this is particularly useful if you want to make some changes to the way flexbox behaves. Example use case is on narrow screen sizes it would allow me to position the image differently to account for the narrow format, this use case being something I want to do on `f-offers` header component. This use case seems very specific so I thought by adding an override class it would still give me the benefits of using the component.

Reply:
Should this not just be a prop that controls the positioning of the image? Depends how specific you think the functionality is I guess but generally I'd try to do this through exposing props rather than classnames.

Reply:
I think for now I will remove until I can figure out a better way of dealing with this, as the stacking on narrow is more important atm. :)